apps/methodologies/bold/rule-processors/credit/rewards-distribution/src/lib/rewards-distribution.processor.spec.ts

```diff
@@ -55,17 +67,36 @@ describe('RewardsDistributionProcessor', () => {
     },
     {
       credit: stubCreditDocument(),
-      errorMessage: 'assert',
+      errorMessage: ruleDataProcessor['ErrorMessage'].INVALID_END_EVENT_VALUE(
+        'documentId',
+        'invalid',
+      ),
       massCertificateAudits: [
         stubMassCertificateAuditDocumentWithResultContent(),
       ],
       masses: [
-        stubMassDocumentWithEndEventValue(
-          String(faker.number.int()) as unknown as number,
-        ),
+        stubMassDocument({
+          externalEvents: [
+            stubDocumentEventWithMetadataAttributes({
+              name: END,
+              value: 'invalid' as unknown as number,
+            }),
```

Comment:
```suggestion
            stubDocumentEventWithMetadataAttributes({
              name: END,
              value: 'invalid' as never,
            }),
```
Not a change request, but a suggestion. We can use `never` instead double casting.